pybossa/repositories/project_repository.py

```diff
@@ -252,17 +252,37 @@ def get_projects_report(self, base_url):
             '''
             )
 
+        sql_worker_details = text(
```

Comment:
Is the subquery needed? Would
```sql
SELECT
  project.id as project_id,
  STRING_AGG(CONCAT('(', t.user_id, ';', w.fullname, ';', w.email_addr, ')'), '|') as workers
FROM
project JOIN task_run t ON project.id = t.project_id
JOIN "user" w ON t.user_id = w.id
GROUP BY project.id;
```
work?

Reply:
Under dev env, i don't see this query returning required data. Another point I noted was (with explain analyze) that this query took close to twice execution time than the query in pr.

Reply:
I missed a `DISTINCT` on the query. Even then it is slower than subquery according to explain plan, so this version should work. The distinct on line 257 should not be needed though.